src/Piling/PilingCoordinator.cs

```diff
@@ -16,6 +19,9 @@ private PilingCoordinator(ISharedParameterManager spManager)
             _variableLoad = PilingParameter.VariableLoad(spManager);
             _verticalWindLoad = PilingParameter.VerticalWindLoad(spManager);
             _horizontalWindLoad = PilingParameter.HorizontalWindLoad(spManager);
+
+            _warnIsFamilyDocumentId = new FailureDefinitionId(new Guid("2c644284-59fe-4f5c-b8b3-e89977af7d15"));
+            _warnIsFamilyDocumentDef = FailureDefinition.CreateFailureDefinition(_warnIsFamilyDocumentId, FailureSeverity.Warning, "Piling coordinates not maintained in family document.");
```

Comment:
Dont hardcode strings - especially relevant for this as we may pacakge for appstore.